main.tf

```diff
@@ -79,11 +85,69 @@ resource "aws_rds_cluster" "this" {
   tags = var.tags
 }
 
+resource "aws_rds_cluster" "this-s3-restore" {
```

Comment:
Please merge this resource with the one we already have and use.

In Terraform 0.12+ there is no need to have 2 resources like this anymore, because we can have `dynamic` blocks.